src/sentiment.jl

```diff
@@ -74,10 +78,22 @@ function Base.show(io::IO, s::SentimentAnalyzer)
 end
 
 
-function(m::SentimentModel)(text::Array{T, 1}) where T <: AbstractString
-    return get_sentiment(text, m.weight, m.words)
+function(m::SentimentModel)(text::Array{T, 1}, handle_unknown) where T <: AbstractString
+    return get_sentiment(text, m.weight, m.words, handle_unknown)
 end
 
-function(m::SentimentAnalyzer)(d::AbstractDocument)
-    m.model(tokens(d))
+
+"""
+ ```
+ model = SentimentAnalyzer(doc)
+ model = SentimentAnalyzer(doc, handle_unknown)
+ ```
+ Return sentiment of the input doc in range 0 to 1, 0 being least sentiment score and 1 being
+ the highest:
+  -  doc              = Input Document for calculating document (AbstractDocument type)
+  -  handle_unknown   = A function for handling unknown words. Should return an array (default (x)->[])
+ """
+
+function(m::SentimentAnalyzer)(d::AbstractDocument, handle_unknown::Function = (x)->[])
```

Comment:
```suggestion
function (m::SentimentAnalyzer)(d::AbstractDocument, handle_unknown = x->tuple())
```

Also again `handle_unknown` should be first argument.

don't constrain callables to be subtypes of functions unless dispatching on them,
and it is maginally faster to use `tuple()` than `[]`